themes/doom-ir-black-theme.el

```diff
@@ -0,0 +1,164 @@
+;;; doom-ir-black-theme.el --- port of the original ir_black theme
```

Comment:
Please add ` -*- lexical-bindings: t; no-byte-compile: t; -*-`

This will fix the `Bytecode overflow` errors in the tests.